### DISCUSSION TITLE
Modification of the names of the exports (csv)

### DISCUSSION THREAD

**Discussion by NetboXNG** [netbox-community/netbox#12572] [Help Wanted!]
Hello,

Here is my problem. When I export in .csv from the netbox "Export" option of one of my template exports from the cables, the file that downloads has the name "netbox_cables.csv". I would like it to have the name of my export script.

For example, when I run my export script "Export_cables_volume" I would like the file to have the same name "Export_cables_volume.csv" and not "netbox_cables.csv".

Thank you in advance for your help

![demande_LI](https://github.com/netbox-community/netbox/assets/133349386/63cbd521-78b9-4f63-a735-6f0140886775)

**Comment by NannsIyam** [comment]
Hi [NetboXNG](https://github.com/NetboXNG),

By export script, do you mean Export Template?
If yes, we have same kind of need here.

We would like to have the Export Template generate a file which name would be the Export Template name.
For instance, an Export Template named "List of Site A switches"  would generate a "list_of_site_a_switches.csv" file.

I didn't find any conversation about it. I think it could deserve a pull request.

**Comment by NetboXNG** [comment]
Hello Nannslyam,

Yes, we have the same problem. I'm going to find out how to make a pull request
